### PR TITLE
P-876 add azure-cli

### PR DESCRIPTION
**Pull request by m1iktea**
### Labels
- C1

### CODE REVIEW

**Review by BillyWooo — COMMENTED** [review]
There are some secrets name got changed. Are they also updated accordingly within repo setting? Or what's the new value? prepared?

**Comment by BillyWooo** [comment]
try to provide some PR description.

File: .github/workflows/benchmark-machine.yml [hunk 1]
```diff
@@ -9,14 +9,16 @@ on:
...
+  INSTANCE_HOST: ${{ secrets.BENCHMARK_INSTANCE_HOST }}
   BENCHMARK_SSH_USER: ${{ secrets.BENCHMARK_SSH_USER }}
-  BENCHMARK_SSH_KEYPATH: ${{ secrets.BENCHMARK_SSH_KEYPATH }}
+  BENCHMARK_SSH_KEY: ${{ secrets.BENCHMARK_SSH_KEY }}
 
 jobs:
   ## run the benchmarking remotely
   benchmark-machine:
-    runs-on: jumphost
+    runs-on: jumphost.azure
```
Comment (Kailai-Wang) [inline .github/workflows/benchmark-machine.yml]: As discussed let's try to get rid of the self-hosted runner completely

**Comment by m1iktea** [comment]
Based on previous testing, the Azure authentication token requires manual updating daily. Currently, there are two proposed solutions:

1. Continue using Azure to run benchmark machines without shutting them down, allowing for direct SSH connections.
2. Shift to using AWS for operations, potentially requiring the application for new AWS quota allocation later on.

**Review by Kailai-Wang — APPROVED** [review]
Great, thanks!

**Pull request merged**